Scripts/script-ToTable.yml

```diff
@@ -5,7 +5,8 @@ commonfields:
 name: ToTable
 script: |-
   var headers = args.columns ? args.columns.split(',') : null;
-  return {Type: entryTypes.note, Contents: tableToMarkdown(args.title, args.data, headers), ContentsFormat: formats.markdown};
+  var flattenObject = treeToFlattenObject(args.data);
```

Comment:
I added it recently to allow taking an array in the context and printing it nicely. It's basically a wrapper script for `tableToMarkdown`.